### ISSUE TITLE
Can't paste whole text into OtpInputs

### ISSUE THREAD

**Issue by thepday12** [dsznajder/react-native-otp-inputs#13]
```
 <OtpInputs
                handleChange={setOTP}
                numberOfInputs={4}
                focusedBorderColor="transparent"
              />
```
I create OtpInputs with 4 inputs. I get OTP have 4 character(ex:1234) when i click fill OTP only first input show "1".
<img width="426" alt="screen shot 2019-01-25 at 11 36 06 am" src="https://user-images.githubusercontent.com/3842720/51725559-749a5c80-2095-11e9-8bd9-41fd5bbb8eb4.png">

**Comment by dsznajder** [comment]
I will sit to this one finally through weekend :) I hope i will come up with some idea.

**Comment by dsznajder** [comment]
Released in v2.0.0. Now you can paste your code into input :)
